routes/conversations.js

```diff
@@ -52,8 +54,35 @@ exports.insertConversation = function(req, res) {
         .catch(error => res.status(error.code || 500).json(error));
 };
 
-exports.updateConversation = function(req, res) {
+exports.updateConversation = async function(req, res) {
     const { project_id: projectId, sender_id: senderId } = req.params;
+
+    try {
+        const userUtterances = req.body.events
+            .filter(event => event.event === 'user' && event.text.indexOf('/') !== 0);
```

Comment:
I get that `event.text.indexOf('/') !== 0` should cover the cases, but checking explicitly if the text starts with '/' would make the code more readable